recipe/meta.yaml

```diff
@@ -15,83 +17,188 @@ source:
     - pyproject.patch
 
 build:
-  number: 0
+  number: 1
   skip: true  # [aarch64]
   missing_dso_whitelist:
     - '*/RDKit*dll'  # [win]
 
 requirements:
+  # See https://conda-forge.org/docs/maintainer/knowledge_base/#placing-requirements-in-build-or-host and
+  # https://conda-forge.org/docs/maintainer/knowledge_base/#details-about-cross-compiled-python-packages.
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
+    - {{ stdlib('c') }}
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy                               # [build_platform != target_platform]
+    - postgresql                          # [build_platform != target_platform]
+    - python                              # [build_platform != target_platform]
     - cmake
-    - jom  # [win]
+    - jom   # [win]
     - make  # [unix]
     - pkg-config
   host:
-    - libboost-devel
-    - libboost-python-devel
     - cairo
     - eigen
     - freetype
-    - python
+    - libboost-devel
+    - libboost-python-devel
     - numpy
-    - pillow
     - pandas <2.2
+    - pillow
     - pip
+    - postgresql
+    - python
     - setuptools
     - setuptools_scm >=8
-    # For rdkit-stubs
-    - pybind11-stubgen
-  run:
-    - cairo
-    - python
-    - pillow
-    - pandas
-    - {{ pin_compatible('numpy') }}
-    - pycairo
-    - matplotlib-base
-    - sqlalchemy
-    - reportlab
-
-test:
-  commands:
-    - python -c "import rdkit; assert rdkit.__version__ == '{{ version }}'"
-  imports:
-    - rdkit
-    - rdkit.Avalon
-    - rdkit.Chem
-    - rdkit.Chem.AllChem
-    - rdkit.Chem.rdFreeSASA
-    - rdkit.DataManip
-    - rdkit.Dbase
-    - rdkit.DistanceGeometry
-    - rdkit.ForceField
-    - rdkit.Geometry
-    - rdkit.ML
-    - rdkit.Numerics
-    - rdkit.SimDivFilters
-    - rdkit.VLib
-    - rdkit.VLib.NodeLib
-    - rdkit.Chem.PandasTools
 
 outputs:
-  - name: rdkit
-  - name: rdkit-dev
-    script: install_rdkit_dev.bat  # [win]
+  - name: librdkit
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+        - numpy                               # [build_platform != target_platform]
+        - python                              # [build_platform != target_platform]
+        - cmake
+      host:
+        - cairo
+        - eigen
+        - freetype
+        - libboost-devel
+        - libboost-python-devel
+        - matplotlib-base
+        - numpy
+        - pandas <2.2
+        - pillow
+        - pycairo
+        - python
```

Comment:
That's right; we build everything, including the python bindings, in build.sh and then install only what we need and list all of the run reqs in each package individually.

Reply:
Additional context: AFAICT it would be quite difficult to build what we need in each subpackage; we'd probably need to restructure the upstream CMake configuration to enable piecewise builds.